src/middleware/error-handler.ts

```diff
@@ -0,0 +1,24 @@
+import { NextFunction, Request, Response } from "express";
+import logger from "../utils/logger";
+import * as config from "../config";
+
+const pageNotFound = (req: Request, res: Response) => {
+  return res.status(404).render(config.NOT_FOUND_TEMPLATE, {
+    templateName: config.NOT_FOUND_TEMPLATE
+  });
+};
+
+/**
+ * This handler catches any other error thrown within the application.
+ * Use this error handler by calling next(e) from within a controller
+ * Always keep this as the last handler in the chain for it to work.
+ */
+const errorHandler = (err: Error, req: Request, res: Response, _next: NextFunction) => {
```

Comment:
Are there any jest tests that can be added for this?

Reply:
there is one